_posts/en/newsletters/2019-06-12-newsletter.md

```diff
@@ -0,0 +1,328 @@
+      background][bg hwi])
+
+      The history and current design of the [version 2 P2P transport
+      protocol][xs v2 p2p] was discussed, as was the related
+      countersign protocol (see [#27][countersign blurb]).  Several
+      possible enhancements were mentioned during the discussion.
+      ([More background][bg v2 p2p])
+
+      The [signet][xs signet] idea for a testnet-like block chain where
+      all of the blocks are signed by a trusted party was described.
+      Discussion focused on the various ways to accomplish this, such as
+      by modifying the block header, the block contents, or the P2P
+      protocol.  Also discussed was various ways this would make Bitcoin
+      development easier.  ([More background][bg signet])
+
+      Although just announced this week on the Bitcoin-Dev mailing list,
+      [blind statechains][xs blind state] were discussed.  This mainly
+      consisted of questions and answers for both technical topics and
+      possible applications of the idea.
+
+- **Proposal to override some BIP125 RBF conditions:** [BIP125][]
+  opt-in Replace-By-Fee (RBF) only allows replacing a transaction with a
+  higher-feerate alternative if the replacement increases the total
+  amount of transaction fee paid in the entire mempool by a certain
+  amount (the incremental feerate).  This stops a cheap
+  bandwidth-wasting Denial-of-Service (DoS) attack against full nodes but
+  makes possible a fairly cheap [transaction pinning][] DoS attack
+  against certain uses of RBF, such as in time-sensitive protocols like
+  LN.
+
+    Several developers have previously attempted to solve this dilemma,
+    with a proposal late last year from Matt Corallo containing a
+    possible solution using CPFP fee bumping ([CPFP carve out][], see
+    [Newsletter #24][]) and suggesting an alternative workaround that
+    tweaks RBF.  Rusty Russell previously [discussed][russell-corallo
+    rbf] this RBF change with Corallo, has further refined it, and now
+    [proposed it][rbf rule 6] to the Bitcoin-Dev mailing as the addition
+    of a single rule to the BIP125 ruleset.  The new rule allows any
+    BIP125 replaceable transaction in the mempool to be fee bumped under
+    two conditions:
+
+    1. The version currently in the mempool is below of the top
+       1,000,000 most profitable vbytes (the next block area)
+
+    2. The replacement version pays a high enough fee that it will be in
+       the next block area
+
+    This would allow any transaction to be fee bumped without
+    consideration of any of that transaction's children, eliminating the
+    problem with pinning.  However, anyone using this rule could reduce
+    the overall amount of fee in the mempool and might be able to use it
+    to excessively waste node bandwidth.  Several people replied to the
+    proposal asking questions and offering analysis about these risks.
+
+- **Presentation: The Next Softfork:** Bitcoin Optech contributor Steve
```

Comment:
I think this section could be shortened slightly, perhaps by giving less details about the 2-of-3 MAST construction. It'd be a shame if people skipped watching the video because your summary was *too* good :)

Reply:
I am comfortable with the current text or a shortened version, whatever you feel is best.

Reply:
I added some words pointing out that @moneyball's slides are easier to follow than the condensed summary of his walkthrough.  Please let me know if that doesn't adequately address the concern.

Reply:
Looks good to me, especially with the shortened coredev section. Thanks Dave.